bridges/AnidexBridge.php

```diff
@@ -3,7 +3,9 @@ class AnidexBridge extends BridgeAbstract {
 
 	const MAINTAINER = 'ORelio';
 	const NAME = 'Anidex';
-	const URI = 'https://anidex.info/';
+	const URI = 'http://anidex.info/'; // anidex.info has ddos-guard so we need to use anidex.moe
```

Comment:
Why switch from https to http?

Reply:
That's due to the change on lines 141-142: because of certificate mismatch on the host which requires to disable certificate validation, the HTTPS connection made to Anidex is insecure. I want the message `Warning : This bridge is not fetching its content through a secure connection` to show on the bridge description, hence the switch in URI.